Source/AliveLibAE/Input.cpp

```diff
@@ -1379,13 +1382,34 @@ EXPORT int Input_Convert_KeyboardGamePadInput_To_Internal_Format_492150()
                             {
                                 pressed_keyboard_keys = keys_down;
                                 input_command_delete_pressed = 1;
+                                ++shoulderButtonsPressedCount; // Added for L2 chanting
+                            }
+                        }
+
+                        if (sGamePadBindings_5C98E0[i] & InputCommands::Enum::eRun) // Added for R1 chanting
+                        {
+                            if ((1 << i) & pButtons)
+                            {
+                                ++shoulderButtonsPressedCount;
+                            }
+                        }
+
+                        if (sGamePadBindings_5C98E0[i] & InputCommands::Enum::eSneak) // Added for R2 chanting
+                        {
+                            if ((1 << i) & pButtons)
+                            {
+                                ++shoulderButtonsPressedCount;
                             }
                         }
                     }
 
-                    if (input_command_c_pressed)
+                    if (shoulderButtonsPressedCount > 1) // Added functionality for PC - chant with any shoulder button combo
+                    {
+                        pressed_keyboard_keys |= InputCommands::Enum::eChant;
+                    }
+                    else if (input_command_c_pressed)
                     {
-                        if (input_command_delete_pressed)
+                        if (input_command_delete_pressed) // Old way of chanting on PC - no longer called - kept for preservation
```

Comment:
if there is any point of this if its dead code?

Reply:
Wasn't sure if anyone needed the old functionality for testing side-by-side, or preservation. A bool could be added to switch between the two. Alternatively, it could just be removed.